free-programming-books.md

```diff
@@ -615,6 +615,7 @@
 * [Getting Real](http://gettingreal.37signals.com)
 * [Google Maps API Succinctly](https://www.syncfusion.com/ebooks/google_maps_api_succinctly) - Mark Lewin
 * [Hacknot: Essays on Software Development](http://www.lulu.com/shop/ed-johnson/hacknot-essays-on-software-development/ebook/product-17544641.html) - Ed Johnson
+* [Hello SDL](https://lazyfoo.net/tutorials/SDL/01_hello_SDL/index.php) - Lazy Foo' Productions
```

Comment:
Could you please remove the redundant part of the URL at the end?
> always prefer the shortest link: http://example.com/dir/ is better than http://example.com/dir/index.html  
> \- https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#guidelines

```suggestion
* [Hello SDL](https://lazyfoo.net/tutorials/SDL/01_hello_SDL/) - Lazy Foo' Productions
```

Reply:
Yeah sure :) But then you may prefer https://lazyfoo.net/tutorials/SDL/ over https://lazyfoo.net/tutorials/SDL/01_hello_SDL/